eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -9,7 +9,9 @@ param (
   $ExitOnError=1,
   $UploadLatest=1,
```

Comment:
Just a reminder these changes will need to be made in the tools repo. Soon this work will be in the language repo once you get your next set of refactorings done.

Reply:
Yh, I will update that PR